auth-web/public/config/configuration.json

```diff
@@ -22,5 +22,7 @@
   "PAY_API_URL": "https://pay-api-dev.apps.silver.devops.gov.bc.ca",
   "PAY_API_VERSION": "/api/v1",
   "LEGAL_API_URL": "https://legal-api-dev.apps.silver.devops.gov.bc.ca",
-  "LEGAL_API_VERSION": "/api/v1"
+  "LEGAL_API_VERSION": "/api/v1",
+  "ONE_STOP_URL": "https://dev.onestop.gov.bc.ca/",
+  "CORPORATE_ONLINE_URL": "https://dev.corponline.gov.bc.ca/"
```

Comment:
If these aren't yet populated in Openshift, ask some Ope guy (Patrick, Richard) to update 1Password accordingly.
And/or add this item to the respective release ticket (so we remember to do it).

Reply:
oh right, i forgot about those changes after the migration!
Will do, thanks 👍

Reply:
Contacted some SRE team members but i can't seem to find a release ticket. 
Just labelled as `Program Increment 10 - All` 🤷

Reply:
Ask James.

Or if the config for all environments is updated now, adding to release ticket isn't needed.

Reply:
Sounds like Richard is already on it, so will confirm and comment back here before resolving and/or merge etc 👍

Reply:
Richard has applied these values to dev and the work order is in for TEST/PROD 😁